### PR TITLE
ADEN-2674 Evolve2 provider integration

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Here is the code with Evolve2 provider (desktop&mobile) which uses our gptHelper like directGpt, remnant, turtle.

Previous PR: https://github.com/Wikia/app/pull/9077
Compare: https://github.com/Wikia/app/compare/ADEN-2629-evolve2...ADEN-2674-evolve2-mobile

### CODE REVIEW

File: extensions/wikia/AdEngine/js/config/mobile.js [hunk 1]
```diff
@@ -2,17 +2,27 @@
...
+	instantGlobals
+) {
 	'use strict';
 
 	var pageTypesWithAdsOnMobile = {
 			'all_ads': true,
 			'corporate': true
-		};
+		},
+		dartEnabled = !instantGlobals || !instantGlobals.wgSitewideDisableGpt;
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/config/mobile.js]: Isn't it confusing that `dartEnabled` depends on `wgSitewideDisableGpt`? How about renaming it to `gptEnabled`?

File: extensions/wikia/AssetsManager/config.php [hunk 2]
```diff
@@ -750,7 +751,9 @@
 		'//extensions/wikia/AdEngine/js/AdEngine2.js',
 		'//extensions/wikia/AdEngine/js/AdLogicPageParams.js',
 		'//extensions/wikia/AdEngine/js/AdTracker.js',
+		'//extensions/wikia/AdEngine/js/DartUrl.js',
```
Comment (nandy-andy) [inline extensions/wikia/AssetsManager/config.php:754]: Oh, what's that... we're using it in `adengine2_desktop_js` as well...

**Comment by nandy-andy** [comment]
One small comment and besides that LGTM :+1:

File: extensions/wikia/AssetsManager/config.php (hunk 2 above)
Comment (fraszczakszymon) [inline extensions/wikia/AssetsManager/config.php:754]: Yup, it's a EvolveHelper's dependency

File: extensions/wikia/AdEngine/js/config/mobile.js (hunk 1 above)
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/config/mobile.js]: :+1:

**Comment by fraszczakszymon** [comment]
`ImageServingCroppingTest.testCropping` has failed - not related to changes

**Comment by jacekjursza** [comment]
LG

**Pull request merged**